src/Utils/hiopOptions.cpp

```diff
@@ -210,6 +210,20 @@ void hiopOptions::registerOptions()
 		      "fixed_var_perturb (default 1e-8)");
   }
 
+  // scaling
+  {
+    vector<string> range(2); range[0]="none"; range[1]="gradient";
+    registerStrOption("scaling_type", "gradient", range,
+		      "The method used for scaling the problem before solving it."
+		      "Setting this option to 'gradient' will scale the problem, guaranteeing the maximum "
+		      "gradient at the initial point is less or equal to scaling_max_grad (default 'none')");
+    
+    registerNumOption("scaling_max_grad", 100, 1e-20, 1e+20,
+		      "The maximum initial gradient allowed in the problem."
+		      "Any gradient that is larger than this positive value will be scaled."
```

Comment:
maybe something like this sound better?

"The problem will be rescaled is the inf-norm of the gradient at the starting point "
"will be larger than the value of this option"

Reply:
sounds better, except for the typo.